lib/geocalc.ex

```diff
@@ -32,6 +32,30 @@ defmodule Geocalc do
     Calculator.distance_between(point_1, point_2)
   end
 
+  @doc """
+  Calculates if a point is within radius of
+  the center of a circle. Return boolean.
+
+  ## Example
+      iex> berlin = [52.5075419, 13.4251364]
+      iex> paris = [48.8588589, 2.3475569]
+      iex> Geocalc.within?(10, paris, berlin)
+      false
+      iex> Geocalc.within?(10, berlin, paris)
+      false
+
+  ## Example
+      iex> san_juan = %{lat: 18.4655, lon: 66.1057}
+      iex> puerto_rico = %{lat: 18.2208, lng: 66.5901}
+      iex> Geocalc.within?(170_000, puerto_rico, san_juan)
+      true
+  """
+  @spec within?(number, Point.t, Point.t) :: boolean()
+  def within?(radius, _center, _point) when radius <= 0, do: false
```

Comment:
please add test for this case
and maybe it would be better to make guard `when radius < 0`
because when you will compare the same city within 0 then it should be true, that is second test we need to add :)

Reply:
Agreed. All done!